### DISCUSSION TITLE
SQLite timestamp comparison?

### DISCUSSION THREAD

**Discussion by intaek-h** [go-jet/jet#346] [Q&A] [answered]
I am making a simple statement that compares created at to a timestamp.
But I just can't find out how in the doc.

```go
import (
	"github.com/go-jet/jet/v2/sqlite"
	. "github.com/intaek-h/part-time-mothers/.gen/table"
)

// I want to get all the rows that have CreatedAt smaller than "2021-01-01"
JobPostings.CreatedAt.LT(sqlite.TIMESTAMP(sqlite.String("2021-01-01 00:00:00"))),
```

Jet is giving me this error saying `jet: no such function: TIMESTAMP` but in vscode, I can `cmd+click` `sqlite.TIMESTAMP()` to see the original code for it.

please help me

**Comment by houtn11** [comment, marked as answer]
Use `DATETIME`, sqlite doesn't support `TIMESTAMP`.

```golang
JobPostings.CreatedAt.LT(sqlite.DATETIME("2021-01-01 00:00:00")),
```

`TIMESTAMP` shouldn't be exposed for sqlite. It is probably a bug.

**Reply by intaek-h** [reply]
@houten11  thank you it worked!
